### PR TITLE
update workflow actions

### PR DESCRIPTION
**Pull request by kalantar**
Signed-off-by: Michael Kalantar <kalantar@us.ibm.com>

### CODE REVIEW

**Review by sriumcp — APPROVED** [review]
LGTM

File: .github/workflows/releasecharts.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ jobs:
           git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
 
       - name: Install Helm
-        uses: azure/setup-helm@v1
+        uses: azure/setup-helm@v3
         with:
           version: v3.7.1
```
Comment (sriumcp) [inline .github/workflows/releasecharts.yaml:27]: Could you please get rid of the `version` and instead use `token: ${{ secrets.GITHUB_TOKEN }}` argument instead? 

***

We can go for the latest stable here and pass the token as documented in the action.

**Review by sriumcp — CHANGES_REQUESTED** [review]
Minor changer requested.

**Review by Alan-Cha — APPROVED** [review]

**Review by sriumcp — APPROVED** [review]
lgtm

**Pull request merged**